.devcontainer/base-dev/base-dev.Dockerfile

```diff
@@ -27,7 +27,8 @@ RUN pip3 install \
     # from local pathfinding
     plotly \
     pyproj \
-    shapely
+    shapely \
+    flask
```

Comment:
Make the dependencies alphabetical please, but no need to rebuild after

Reply:
done